distributed/deploy/tests/test_local.py

```diff
@@ -1004,3 +1005,13 @@ def test_client_cluster_synchronous(loop):
     ) as cluster:
         async with Client(cluster, asynchronous=True) as client:
             assert client.security == cluster.security
+
+
+@pytest.mark.asyncio
+async def test_no_dangilng_asyncio_tasks(cleanup):
+    start = asyncio.all_tasks()
+    async with LocalCluster(asynchronous=True, processes=False):
+        await asyncio.sleep(0.01)
+
+    tasks = asyncio.all_tasks()
+    assert tasks == start
```

Comment:
I'm not very familiar with asyncio, does this mean that if the assert was inside the async with block it would be fail (i.e., more tasks than during start)?

Reply:
Yes.  This is the collection of all ongoing coroutines/tasks in asyncio.  During Dask execution there are several concurrent asyncio tasks.  We want to verify that they all get cleaned up.  

These tests don't fix anything, but they're nice to have around regardless.  I'm still searching for what is going on here.

Reply:
Makes sense, thanks for the explanation!